scapy/contrib/lldp.py

```diff
@@ -282,6 +284,43 @@ def dissection_done(self, pkt):
 
         super(LLDPDU, self).dissection_done(pkt)
 
+    def _check(self):
+        """Overwrited by LLDPU objects"""
+        pass
+
+    def post_dissect(self, s):
+        self._check()
+        return super(LLDPDU, self).post_dissect(s)
+
+    def do_build(self):
+        self._check()
+        return super(LLDPDU, self).do_build()
+
+class _LLDPidField(StrLenField):
```

Comment:
Could you use a linter on this class and add docstring when necessary ?

Reply:
Updated!